.travis.yml

```diff
@@ -4,10 +4,17 @@ node_js:
   # TODO: stable (v13.0.1 at the time) breaks sha3 package compilation
   # Sticky v12
   - '12'
-cache: 'yarn'
+cache:
+  yarn: true
+  directories:
+    - ~/.cache
```

Comment:
You sure we need this,`.cache` in home folder?
I've seen [`$HOME/.cache`](https://docs.travis-ci.com/user/caching/#yarn-cache) somewhere before
Also this cache setup looks like a blanket cache of everything. Did you see it recommended somewhere?

Reply:
https://docs.cypress.io/guides/getting-started/installing-cypress.html#Binary-cache

I was considering consolidating all the caches in this folder if this doesn't add any troubles (like not existing intermediate dir `.cache/`)

Reply:
Also https://docs.cypress.io/guides/guides/continuous-integration.html#Example-travis-yml-config-file

Reply:
On, you convinced me